docs/feature_space_cadet_shift.md

```diff
@@ -1,17 +1,30 @@
 # Space Cadet Shift: The Future, Built In
 
-Steve Losh described the [Space Cadet Shift](http://stevelosh.com/blog/2012/10/a-modern-space-cadet/) quite well. Essentially, when you tap Left Shift on its own, you get an opening parenthesis; tap Right Shift on its own and you get the closing one. When held, the Shift keys function as normal. Yes, it's as cool as it sounds.
+Steve Losh described the [Space Cadet Shift](http://stevelosh.com/blog/2012/10/a-modern-space-cadet/) quite well. Essentially, when you tap Left Shift on its own, you get an opening parenthesis; tap Right Shift on its own and you get the closing one. When held, the Shift keys function as normal. Yes, it's as cool as it sounds, and now even cooler supporting Control and Alt!
 
 ## Usage
 
+Firstly, enable Space Cadet in your keymap's Makefile.
+
+```make
+SPACE_CADET_ENABLE = yes
+```
+
+Then do one of the following:
 Replace the Left Shift key in your keymap with `KC_LSPO` (Left Shift, Parenthesis Open), and Right Shift with `KC_RSPC` (Right Shift, Parenthesis Close).
```

Comment:
Perhaps make these three sentences into a list, the way it gets rendered now they all run together in a single paragraph.